packages/pharos/src/components/pagination/pharos-pagination.ts

```diff
@@ -143,12 +196,98 @@ export class PharosPagination extends ScopedRegistryMixin(PharosElement) {
     return nothing;
   }
 
+  private _renderLastLink(): TemplateResult | typeof nothing {
+    if (this.variant === 'input' && this.currentPage < this.totalPages) {
+      return html`
+        <pharos-link
+          class="pagination__link last"
+          subtle
+          flex
+          href=""
+          aria-label="Go to last page"
```

Comment:
I _think_ because we have the `pagination__visually-hidden` span we don't need the aria-label here